gobblin-utility/src/main/java/gobblin/util/JobLauncherUtils.java

```diff
@@ -76,8 +76,8 @@ public static String newMultiTaskId(String jobId, int sequence) {
    * list by checking each element of the given list, and seeing if it is an instance of {@link MultiWorkUnit}. If it is
    * then it iterates through all the {@link WorkUnit}s returned by {@link MultiWorkUnit#getWorkUnits()} and adds them
    * to the flattened list. If not, then it simply adds the {@link WorkUnit} to the flattened list.
-   * @param workUnits is a {@link List} containing either {@link State}s or {@link MultiWorkUnit}s.
-   * @return a {@link List} of {@link State}s.
+   * @param states is a {@link List} containing either {@link State}s or {@link MultiWorkUnit}s
+   * @return a {@link List} of {@link State}s
    */
   public static List<State> flattenWorkUnits(List<? extends State> states) {
```

Comment:
This method doesn't seem to support a MultiWorkUnit containing another MultiWorkUnit as child. Probably should clarify this in the javadoc.